zone/spell_effects.cpp

```diff
@@ -5239,13 +5289,16 @@ int32 Mob::CalcFocusEffect(focusType type, uint16 focus_id, uint16 spell_id, boo
 	10/11 SE_LimitCastingSkill:
 	12/13 SE_LimitSpellClass:
 	14/15 SE_LimitSpellSubClass:
+	16/17 SE_FFItemCLass:
 	Remember: Update MaxLimitInclude in spdat.h if adding new limits that require Includes
 	*/
 
 	for (int i = 0; i < EFFECT_COUNT; i++) {
 
-		switch (focus_spell.effect_id[i]) {
+		Shout("Check SPA %i", focus_spell.effect_id[i]);
```

Comment:
Debug message here.

Reply:
oops thought I fixed that, must have forgot to hit push. resolved